### DISCUSSION TITLE
Stop at 11%

### DISCUSSION THREAD

**Discussion by chiaersssss** [Chia-Network/chia-blockchain#6594] [Support]
win10 

![f6db637c764036fab1018298001c5ec](https://user-images.githubusercontent.com/83908338/121154353-7e454900-c879-11eb-9f62-fddc79eaddeb.jpg)

**Comment by loppefaaret** [comment]
It can be a sign of problems with memory. Try a memtest, or just lowering memory settings, and see if that helps

**Comment by chiaersssss** [comment]
thanks  my memory are ddr3 4Gx2  1600  ,you 's meaning is  changes  1333?

**Reply by loppefaaret** [reply]
That could be worth a try, yes. Or a memtest run

**Comment by chiaersssss** [comment]
test over 108%  0 coverage 0, what can I do please ?

**Reply by loppefaaret** [reply]
Then it might not be a memory issue after all

**Comment by BrandtH22** [comment]
@chiaersssss , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
